### ISSUE TITLE
Feature Request/ Possible New App Request

### ISSUE THREAD

**Issue by Spenser715** [AntonyCorbett/JwlMediaWin#8]
We are having an interesting problem at our KH. We have a setup that uses SDI to send signal from the Media computer to the Media displays. For some reason when there is a full black scene/transition in say a video or between media, the TV thinks it lost signal and shows the Input information. We believe if there is some way to say put an overlay of even just 1 pixel at the edge of the screen (where nooone would even notice it) this would make it not be a full black screen and stop the input screen from flashing. I'm not sure if this is something that could be added to this app or if this could be a new app or if you even know of another app or other solution to this, but I figured it would at least be worth checking. Thanks in advance and for all your hard work on these apps. We love them.

**Comment by LvDaniel** [comment]
@Spenser715 is this happening in the middle of a video if at a point a black scene appears for let's say 1 second?
Did you encounter this while using SoundBox?
We're also using SDI signal in out setup. From PC we're getting out through HDMI, we use a HDMI 2 SDI converter which is located on the AV desk and from there SDI signal goes to TVs ~20m which have SDI input.
We haven't encounter this issue and we're having this setup implemented in 4 halls.
But your setup might be different from ours.

**Comment by AntonyCorbett** [comment]
@Spenser715 I would suspect the HDMI / SDI conversion hardware. Perhaps the best way to isolate is to swap out parts of the kit in turn. You don't say if the problem affects both TVs or just one. If it's just one then you can assume a faulty TV / configuration or a faulty cable. If both TVs are affected start by hooking up a small PC monitor to the PC's HDMI out. If that works, move it to the far end of the cable after HDMI/SDI conversion, etc.

**Comment by blazer2004** [comment]
^ Completely agree. SDI with halfway decent connections/wiring/hardware should not do this, no matter what video is being played. Double check your BNC fittings. What brand are your converters, Monoprice? If so consider upgrading to Blackmagic Design micro converters/distribution amp. If there is a weak link in your SDI system, even a LED lighting system can play games with poor quality converters and distribution amps.

**Comment by Spenser715** [comment]
Thank you all for your responses. I was told the equipment we have is what LDC recommends, We will go through and check all the hardware. Since none of you are having this issue, do any of you happen to know model numbers for your parts that I can compare and see if maybe we can upgrade something if we need to.

**Comment by blazer2004** [comment]
Just remembered, check your refresh rate under display settings. Both the computer monitor and the TVs need to be set at 60hertz or higher, if available.

**Comment by Blann** [comment]
I'm not sure if this is relevant, but we experienced something somewhat similar whenever we started a video in JW Library – the screen would flicker as if it had lost input momentarily. The way we solved it was to change a setting in the Nvidia Control Panel (accessed by right-clicking on the desktop) that told the graphics driver what type of media was being displayed on the screen. I can't remember exactly where it is but it's under the 'Video' section – it needs to be set to 'Full-screen video' rather than any automatic setting. Seems like the auto setting was switching modes when it detected video playback and this was causing the input to drop out momentarily.

Again not sure it's relevant, but just thought I'd mention it.

**Comment by Spenser715** [comment]
Thank you. I will check that and see if that helps.

**Comment by Spenser715** [comment]
Thank you everyone for all your comments. We believe we found a workaround that fixed the issue. We went into the graphics card settings and were checking all the available options. one section was display themes (theater, bright, vibrant, high contrast) and we chose one that sounded like it would add just a little brightness to the screen. And after waiting a week to see if we still had the issue we believe we fixed it.
